test/runtime/kvstore.go

```diff
@@ -68,45 +68,48 @@ var _ = PDescribe("RuntimeKVStoreTest", func() {
 		vm.CloseSSHClient()
 	})
 
-	It("Consul KVStore", func() {
-		ctx, cancel := context.WithCancel(context.Background())
-		defer cancel()
-		By("Starting Cilium with consul as kvstore")
-		vm.ExecInBackground(
-			ctx,
-			"sudo cilium-agent --kvstore consul --kvstore-opt consul.address=127.0.0.1:8500 --debug")
-		err := vm.WaitUntilReady(helpers.CiliumStartTimeout)
-		Expect(err).Should(BeNil())
-
-		By("Restarting cilium-docker service")
-		vm.Exec("sudo systemctl restart cilium-docker")
-		helpers.Sleep(2)
-		containers(helpers.Create)
-		vm.WaitEndpointsReady()
-		eps, err := vm.GetEndpointsNames()
-		Expect(err).Should(BeNil(), "Error getting names of endpoints from cilium")
-		Expect(len(eps)).To(Equal(1), "Number of endpoints in Cilium differs from what is expected")
-	})
-
-	It("Etcd KVStore", func() {
-		ctx, cancel := context.WithCancel(context.Background())
-		defer cancel()
-		By("Starting Cilium with etcd as kvstore")
-		vm.ExecInBackground(
-			ctx,
-			"sudo cilium-agent --kvstore etcd --kvstore-opt etcd.address=127.0.0.1:4001 2>&1 | logger -t cilium")
-		err := vm.WaitUntilReady(helpers.CiliumStartTimeout)
-		Expect(err).Should(BeNil(), "Timed out waiting for VM to be ready after restarting Cilium")
-
-		By("Restarting cilium-docker service")
-		vm.Exec("sudo systemctl restart cilium-docker")
-		helpers.Sleep(2)
-		containers(helpers.Create)
-
-		vm.WaitEndpointsReady()
-
-		eps, err := vm.GetEndpointsNames()
-		Expect(err).Should(BeNil(), "Error getting names of endpoints from cilium")
-		Expect(len(eps)).To(Equal(1), "Number of endpoints in Cilium differs from what is expected")
+	SkipContextIf(helpers.SkipQuarantined, "KVStore tests under quarantine", func() {
```

Comment:
Can't we use skip the tests in `BeforeAll` instead? The extra indentation just to disable the tests will make `git blame` harder...

Reply:
@pchaigno I wanted to skip whole `Describe` block at first, but ginkgo doesn't allow that. I would like to have one preferred way to put tests under quarantine, and I think the most best way to do it is by explicitly changing the test/context signature.

`git blame -w` can help with whitespace changes.

Alternatively, I can change signatures of individual test cases in this suite, would that be better in your opinion?

Reply:
TIL `git blame -w` :smiley: 
So I'm fine with this change as is given that wonderful option.

Reply:
It even works with fugitive vim plugin!